region/brinstar/pink.json

```diff
@@ -1072,7 +1072,17 @@
                 {
                   "name": "Base",
                   "notable": false,
-                  "requires": []
+                  "requires": [
+                    {"or": [
+                      "Grapple",
+                      "h_canFly",
+                      "canWalljump",
+                        {"and": [
+                          "SpeedBooster",
+                          "HiJump"
+                        ]}
```

Comment:
A SpringBall Jump (with HiJump) works here. You can also do a non-HiJump SpringBall Jump if starting from a frozen Sidehopper (doesn't work on the ground, but the apex of a small hop works).

There's also a ~24-frame shinespark you can charge in-room at node 3's runway. Can also be extended by charging in the adjacent room (with a `canComeInCharged from node 3, with 180 frames remaining, and an inRoomPath of [3, 13]).